### PR TITLE
Add edit page

### PR DESCRIPTION
**Pull request by goncalopc96**

### CODE REVIEW

**Comment by pawel-krysiak** [comment]
🆗

**Comment by donbonifacio** [comment]
I added a new config and I had this result:
screen shot 2016-06-21 at 15 05 58

File: librato_chart_sender/forms.py [hunk 1]
```diff
@@ -43,17 +42,25 @@ def __validate_librato_email(self, email):
...
+            return valid
 
     def __validate_integers(self, integers):
-        valid = True
-        for integer in integers:
-            if not self.__validate_integer(integer):
-                self.errors.append('{integer} is not a valid chart number.'.format(integer=integer))
-                valid = False
-        return valid
+        if not integers:
```
Comment (donbonifacio) [inline librato_chart_sender/forms.py:57]: What is integers?

File: librato_chart_sender/urls.py [hunk 2]
```diff
@@ -5,6 +5,6 @@
 urlpatterns = [
     url(r'^$', views.index, name='index'),
     url(r'^config/new/$', views.config_new, name='config-new'),
-    # url(r'^config/new/$', views.config_new_save, name='config-new'),
-    # url(r'^config/temp.html', views.config_temp, name='config-temp'),
+    url(r'^config/edit/(?P<config_id>[0-9]+)/$', views.config_edit, name='config-edit'),
```
Comment (donbonifacio) [inline librato_chart_sender/urls.py:8]: Edit and delete are not working.

File: librato_chart_sender/forms.py (hunk 1 above)
Comment (pawel-krysiak) [inline librato_chart_sender/forms.py:57]: any list of numbers

**Comment by donbonifacio** [comment]
I installed it and was able to do suff. :)
Congrats.

LGTM 🆗

**Comment by donbonifacio** [comment]
👍 

I actually tried the proper branch and everything is working. 🙍

File: librato_chart_sender/forms.py (hunk 1 above)
Comment (donbonifacio) [inline librato_chart_sender/forms.py:57]: Ok, to validate the charts id right?

**Pull request merged**